nix/overlay.nix

```diff
@@ -3,7 +3,11 @@ let
 in
 self: super:
 {
-  gpgme = (static super.gpgme);
+  gpgme = (static super.gpgme).overrideAttrs (x: {
+    # Drop the --enable-fixed-path:
+    # https://github.com/nixos/nixpkgs/blob/9a79bc99/pkgs/development/libraries/gpgme/default.nix#L94
+    configureFlags = self.lib.lists.drop 1 x.configureFlags;
```

Comment:
This looks fine, but a bit brittle. Can the order of the included flag change? Maybe.

Reply:
Yeah it can, we probably have to make it a bit more robust if the manual test succeeds.

Reply:
Now it should be better :)

Reply:
Now, that looks solid!